src/test/java/io/openliberty/tools/intellij/it/UIBotTestUtils.java

```diff
@@ -2540,11 +2558,79 @@ public static void findWelcomeFrame(RemoteRobot remoteRobot) {
         } catch (WaitForConditionTimeoutException e) {
             // If the welcome frame is not found then there is a project loaded.
             // Close the editor files and close the project to get back to the welcome frame.
-            UIBotTestUtils.closeAllEditorTabs(remoteRobot);
+            UIBotTestUtils.runActionFromSearchEverywherePanel(remoteRobot, "Close All Tabs", 3);
             UIBotTestUtils.closeProjectView(remoteRobot);
             UIBotTestUtils.closeProjectFrame(remoteRobot);
             TestUtils.sleepAndIgnoreException(30); // takes about 15s to render the whole Welcome page including the Open button on Mac.
             UIBotTestUtils.validateProjectFrameClosed(remoteRobot);
         }
     }
+
+    /**
+     * Clicks on the main menu button.
+     *
+     * @param remoteRobot The RemoteRobot instance.
+     */
+    public static void clickOnMainMenu(RemoteRobot remoteRobot) {
+        String fileName = "Main Menu";
+        ProjectFrameFixture projectFrame = remoteRobot.find(ProjectFrameFixture.class, Duration.ofSeconds(10));
+
+        try {
+            // Clicking on the left toolbar ensures that the Main Menu button is active.
+            String xPath = "//div[@accessiblename='Left Toolbar' and @class='ToolWindowLeftToolbar']";
+            ComponentFixture actionButton = projectFrame.getActionButton(xPath, "10");
+            actionButton.click();
+            // Clicking on the Main Menu
+            String xPathMainMenu = "//div[@accessiblename='" + fileName + "' and @class='ActionButton']";
+            ComponentFixture actionButtonMainMenu = projectFrame.getActionButton(xPathMainMenu, "10");
+            actionButtonMainMenu.click();
+
+        } catch (WaitForConditionTimeoutException e) {
+            // Main Menu button is not clicked, nothing to do
+        }
+    }
+
+    /**
+     * Right-clicks on a file tab and selects "Close" to close the file.
+     *
+     * @param remoteRobot The RemoteRobot instance.
+     * @param fileName    The name of the file tab to be closed.
+     */
+    public static void rightClickCloseOnFileTab(RemoteRobot remoteRobot, String fileName) {
```

Comment:
Also doesn't seem like a file name from the description of the param. fileTabName perhaps?